gdnative-core/src/nativescript/init/property/hint.rs

```diff
@@ -116,30 +119,31 @@ where
 /// ```
 #[derive(Clone, Eq, PartialEq, Debug, Default)]
 pub struct EnumHint {
-    values: Vec<String>,
+    values: HashMap<String, i32>,
+}
+
+impl From<HashMap<String, i32>> for EnumHint {
+    #[inline]
+    fn from(values: HashMap<String, i32>) -> Self {
+        EnumHint { values }
+    }
 }
```

Comment:
I would rather add a method in `EnumHint`, e.g. the suggested `with_numbers()` taking tuples. It allows to create `EnumHint` from a literal syntax and is consistent with the existing `new()` constructor -- there's no `From` implementation for `Vec<String>` either.

Construction could look as follows:
```rs
EnumHint::with_numbers(vec![
    ("one".into(), 1),
    ("two".into(), 2),
    ("seven".into(), 7),
])
```